socless/events.py

```diff
@@ -189,7 +189,7 @@ def to_dict(self):
         return {"event": self.event.__dict__, "metadata": self.metadata.to_dict()}
 
     @property
-    def as_event_table_item(self):
+    def to_EventTableItem(self) -> EventTableItem:
```

Comment:
I think I see the spirit behind the UpperCamelCase notation, but stick with the `to_event_table_item` convention. It doesn't lose meaning since the function is already typehinted with `EventTableItem`

Reply:
`as_event_table_item` does also seem clearer actually since we're "returning it as a format". `convert_to_event_table_item` also makes sense. 

`to_EventTableItem` carries some vagueness to it e.g "Is it sending it `to` something?"